infra/build/functions/build_lib.py

```diff
@@ -75,7 +75,7 @@
                    supported_architectures=['x86_64']),
     'centipede':
         EngineInfo(upload_bucket='clusterfuzz-builds-centipede',
-                   supported_sanitizers=['address'],
+                   supported_sanitizers=['address', 'none'],
```

Comment:
I'm not sure how you plan on doing centipede support but I think this is probably a mistake. Or is the plan for ClusterFuzz to download multiple builds (sanitized and not) for each round of fuzzing?

Reply:
Yes, CF will download multiple builds. This is also more friendly to disk space usage (because we can download, extract the required files, and delete and rinse/repeat) rather than bundling everything in a single huge zip.